docs/intro/overrides.rst

```diff
@@ -100,42 +129,156 @@ The code above declares that:
 
 .. tip::
 
-    The URL patterns declared in the ``@handle_urls`` annotation can still be
+    The URL patterns declared in the ``@handle_urls`` decorator can still be
     further customized. You can read some of the specific parameters in the
     :ref:`API section <api-overrides>` of :func:`web_poet.handle_urls`.
 
+Item Class
+~~~~~~~~~~
+
+An alternative approach for the Page Object Overrides example above is to specify
+the returned Item Class. For example, we could change the previous example into
+the following:
+
+
+.. code-block:: python
+
+    from web_poet import handle_urls, WebPage, Returns
+
+
+    class GenericProductPage(WebPage, Returns[Product]):
+        def to_item(self) -> Product:
+            return Product(product_title=self.css("title::text").get())
+
+
+    @handle_urls("example.com", to_return=Product)
+    class ExampleProductPage(WebPage):
+        def to_item(self) -> Product:
+            ...  # more specific parsing
+
+
+    @handle_urls("anotherexample.com", to_return=Product, exclude="/digital-goods/")
+    class AnotherExampleProductPage(WebPage):
+        def to_item(self) -> Product:
+            ...  # more specific parsing
+
+
+    @handle_urls(["dualexample.com/shop/?product=*", "dualexample.net/store/?pid=*"], to_return=Product)
```

Comment:
example.com is a special domain for documentation purposes, anotherexample.com and dualexample.com are not (e.g. they could be registered and point to NSFW content in the future, making it akward that we have them here). You could instead use the `.example` top-level domain, e.g. `some.example`, `another.example`, `dual.example`.

Reply:
Good idea, that hasn't occurred to me. Yep, let's sanitize the URLs in the docs and tests. https://github.com/scrapinghub/web-poet/pull/84/commits/0626e578b73843c11ec306a92946ee5d3bf78040